app/components/home-page/org-side-menu/index.ts

```diff
@@ -48,6 +49,14 @@ export default class HomePageSideMenuComponent extends Component {
         route: 'authenticated.app-monitoring',
         query: { app_offset: 0 },
       },
+      {
+        label: 'SBOM',
+        icon: 'receipt-long',
+        route: 'authenticated.sbom.apps',
+        query: { app_offset: 0 },
+        currentWhen:
```

Comment:
Will `authenticated.sbom` not cover all nested sbom routes?

Reply:
no it is kind of exact match by default